networkx/drawing/layout.py

```diff
@@ -1111,6 +1112,219 @@ def multipartite_layout(G, subset_key="subset", align="vertical", scale=1, cente
     return pos
 
 
+def estimate_factor(n, swing, traction, speed, speed_efficiency, jitter_tolerance):
```

Comment:
`estimate_factor` is a private function and we name them starting with a single underscore. `_estimate_factor`.

Reply:
In order to prevent bloat in the namespace, I inlined this function inside the function.

Reply:
> In order to prevent bloat in the namespace, I inlined this function inside the function.

This doesn't appear to be true as this is still here - is the PR on the right commit?